storage/tables/data-types/index.md

```diff
@@ -13,7 +13,8 @@ example, the transformation [`COPY` mapping](/transformations/snowflake/#load-ty
 the transformations. Also, some writers, e.g., the [Snowflake writer](/components/writers/database/snowflake/) use
 the table metadata to [pre-fill the table columns](/components/writers/database/snowflake/#table-configuration) configuration for you.
 
-Even if a data type is available for a column, that column is always stored as text --- keep this in mind
+Even if a data type is available for a column, storage always creates internally all columns for table as text not null and null-able values are converted to empty strings (except for Exasol where everything is null). 
+This behavior can be changed in certain cases with feature [native datatypes](#native-datatypes). Keep this in mind

```

Comment:
To `Keep this in mind...` bych dal na další řádky ono sa to víc hodilo k tomu předchozímu odstavci. 
Vlastně bych to prohodil a `This behavior can be changed...` dal až za to. Sorry jak to GH rozdělil tak sem si nevšiml že to pokračuje takto.

Reply:
done, prihodil jsem jeste obrazek